components/Navigation.tsx

```diff
@@ -0,0 +1,30 @@
+import NavigationMenu from './NavigationMenu';
+import styled from 'styled-components';
+import Link from 'next/link';
+
+export default function Navigation() {
```

Comment:
One suggestion for composing the components:
In my eyes this component is more a 'Header' than a 'Navigation'. A header can include a navigation but a navigation normally does not include the app title. Since you already use the `<header>` element I recommend to rename the function and the file as well.